### PR TITLE
Register container (Closes #91)

### LINKED ISSUE #91: Create registration container

### PR DESCRIPTION
**Pull request by leestanley**
These reducers are very jank I am very stuck 😕

### CODE REVIEW

**Comment by eminguyen** [comment]
> Not done yet - made a pr for current progress - will return later tonight

Good luck with google!!

File: src/actions/registerAction.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { REGISTER_USER } from './types';
+
+export const registerAccount = (values) => dispatch => {
+  // TODO - Log in the user here.
+  console.log("meow")
```
Comment (eminguyen) [inline src/actions/registerAction.js]: console.log the values to see if they print out here. If they do, we can just submit a post request with the values

File: src/reducers/RegisterReducer.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { REGISTER_USER } from '../actions/types';
+
+const RegisterReducer = (action) => {
+  switch (action.type) {
+    // TODO Authentificated should not always be true.
+    case REGISTER_USER:
+      return {
+        ...state,
+        admin: false,
```
Comment (eminguyen) [inline src/reducers/RegisterReducer.js]: admin and auth should be in auth reducer only, i think what ucla does with registered / user works

File: src/containers/RegisterPage.jsx [hunk 3]
```diff
@@ -11,3 +11,4 @@ const RegisterPage = () => {
 };
 
 export default RegisterPage;
+
```
Comment (eminguyen) [inline src/containers/RegisterPage.jsx]: is this line supposed to be here

File: src/reducers/RegisterReducer.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  error: null,
+};
+
+const RegisterReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case REGISTER_USER:
+      return {
+        ...state,
+        registeredSuccess: true,
+        register: action.payload,
```
Comment (eminguyen) [inline src/reducers/RegisterReducer.js]: set error to false if user is registered

File: src/containers/RegisterPage.jsx (hunk 3 above)
Comment (leestanley) [reply src/containers/RegisterPage.jsx]: yo my prettier is wack

File: src/reducers/RegisterReducer.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+      };
+    case REGISTER_FAIL:
+      return {
+        ...state,
+        error: true,
```
Comment (eminguyen) [inline src/reducers/RegisterReducer.js]: set registerSuccess to be false. Also, what component do we use registerSuccess for? I guess this should be used to update the register form to say register failed right? For now, we can just leave it out. We can work together tomorrow to implement some component for it once we're hooked up to the backend

File: src/reducers/RegisterReducer.js [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+  registerSuccess: false,
+  error: null,
+};
+
+const RegisterReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case REGISTER_USER:
+      return {
+        ...state,
+        registeredSuccess: true,
```
Comment (eminguyen) [inline src/reducers/RegisterReducer.js]: BRO it's registerSuccess not registeredSuccess

File: src/actions/registerActions.js [hunk 7]
```diff
@@ -0,0 +1,14 @@
+import { REGISTER_USER } from './types';
+import { replace } from 'connected-react-router';
+
+export const registerAccount = (values) => dispatch => {
+  console.log(values)
```
Comment (eminguyen) [inline src/actions/registerActions.js]: Remove console.log

File: src/reducers/RegisterReducer.js (hunk 5 above)
Comment (leestanley) [reply src/reducers/RegisterReducer.js]: Register success should be hooked up to the /authenticate-email component I made

Comment (leestanley) [reply src/reducers/RegisterReducer.js]: https://github.com/acmucsd/membership-portal-ui/pull/96

**Pull request merged**